src/test/java/com/training/baseTest/FlipkartBaseTest.java

```diff
@@ -0,0 +1,34 @@
+package com.training.baseTest;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.util.concurrent.TimeUnit;
+
+public class FlipkartBaseTest {
```

Comment:
This base test will be used also for other web tests other than flipkart. So rename this to WebBaseTest